apps/meteor/tests/e2e/omnichannel/omnichannel-take-chat.spec.ts

```diff
@@ -16,7 +18,7 @@ test.describe('omnichannel-takeChat', () => {
 		await Promise.all([
 			await api.post('/livechat/users/agent', { username: 'user1' }).then((res) => expect(res.status()).toBe(200)),
 			await api.post('/settings/Livechat_Routing_Method', { value: 'Manual_Selection' }).then((res) => expect(res.status()).toBe(200)),
-			await api.post('/settings/Livechat_enabled_when_agent_idle', { value: false }).then((res) => expect(res.status()).toBe(200)),
+			await api.post('/settings/Livechat_enabled_when_agent_idle', { value: true }).then((res) => expect(res.status()).toBe(200)),
```

Comment:
Restore this one to the previous value after your tests.

Reply:
The default value is true on this 🤔

Reply:
This is to ensure that the status is correct before starting the test.